packages/dev-kit-italia/stories/components/it-back.stories.ts

```diff
@@ -0,0 +1,212 @@
+
+Questo metodo consente di navigare all'indietro di una singola entry nello stack della cronologia del browser, replicando il comportamento del pulsante "Indietro" nativo del browser.
+
+\`\`\`javascript
+const backButton = document.getElementById('backButton');
+backButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  window.history.back();
+});
+
+\`\`\`
+Gli esempi seguenti implementano questa funzionalità nel loro codice di esempio.
+
+<br/>
+
+        `.trim(),
+      },
+    },
+  },
+  render: () => html`<div class="hide-preview"></div>`,
+};
+
+export const Link: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: `
+
+Quando si utilizza un link (tag \`<a>\`) è necessario aggiungere un valore all'attributo href perché il link sia correttamente navigabile via tastiera.
+
+Utilizzare la classe \`.go-back\` per applicare lo stile appropriato.
+
+
+\`\`\`
+        `.trim(),
+      },
+    },
+  },
+  render: () => html`
+    <a
+      href="#"
+      class="go-back"
+      @click=${(e: Event) => {
+        e.preventDefault();
+        window.history.back();
+      }}
+    >
+      <it-icon name="it-arrow-left" size="sm" color="primary" class="me-2"></it-icon>
+      <span>Torna indietro</span>
+    </a>
+  `,
+};
+
+export const Pulsante: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: `
+Nel caso dell'utilizzo di un pulsante è possibile utilizzare sia \`<button>\` standard che il componente \`<it-button>\`.
+
+Utilizzare la classe \`.go-back\` per applicare lo stile appropriato.
+
+
+\`\`\`
+        `.trim(),
+      },
+    },
+  },
+  decorators: [
+    (story) => html` <div class="d-flex gap-3 flex-wrap"><!-- Inizio esempi -->${story()}<!-- Fine esempi --></div> `,
+  ],
+  render: () => html`
+    <button
+      type="button"
+      class="btn btn-primary go-back"
+      @click=${() => {
+        window.history.back();
+      }}
+    >
+      <it-icon name="it-arrow-left" size="sm" color="white" class="me-2"></it-icon>
+      <span>Torna indietro</span>
+    </button>
+    <button
+      type="button"
+      class="btn btn-primary go-back"
+      @click=${() => {
+        window.history.back();
+      }}
+    >
+      <it-icon name="it-arrow-up" size="sm" color="white" class="me-2"></it-icon>
+      <span>Torna indietro</span>
+    </button>
+    <it-button
+      variant="primary"
+      class="go-back"
+      @click=${() => {
+        window.history.back();
+      }}
+    >
+      <it-icon name="it-arrow-left" size="sm" color="white" slot="prefix"></it-icon>
+      <span>Torna indietro</span>
+    </it-button>
+    <it-button
+      variant="primary"
+      class="go-back"
+      @click=${() => {
+        window.history.back();
+      }}
+    >
+      <it-icon name="it-arrow-up" size="sm" color="white" slot="prefix"></it-icon>
+      <span>Torna indietro</span>
+    </it-button>
```

Comment:
Qui non sta funzionando l'icona nel button, mentre negli esempi sotto con solo icona sì.
Link alla preview in cui non li vedo: https://design-web-components-git-it-back-dip-trasformazione-digitale.vercel.app/?path=/story/componenti-navigazione-torna-indietro--pulsante

Forse per via dello slot per qualche motivo? Non ho ricontrollato il codice al momento.